src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -328,19 +330,81 @@ private void InsertEntry(NodeSearchElementViewModel entry, IEnumerable<string> c
             var nameStack = new Stack<string>(categoryNames.Reverse());
             var target = libraryRoot;
             bool isRoot = true;
+            string path = "";
```

Comment:
After you have renamed `AddToFullPath`, it'll make sense if we rename this to `fullyQualifiedName`.